packages/hub/src/components/EntityInfo.tsx

```diff
@@ -1,16 +1,28 @@
 import { FC } from "react";
 
 import { UsernameLink } from "./UsernameLink";
+import { modelRoute } from "@/routes";
+import Link from "next/link";
 
 // works both for models and for definitions
 export const EntityInfo: FC<{ username: string; slug: string }> = ({
   username,
   slug,
 }) => {
   return (
-    <div>
-      <span className="text-xl font-bold">{slug}</span> by{" "}
-      <UsernameLink username={username} />
+    <div className="flex justify-between w-full">
```

Comment:
`justify-between w-full` is technically correct, but it makes me think this component should be deleted; or split into two (one for entity name, one for "by username"), if we want to reuse styles on model and definition pages.

Reply:
Yea, I think we'll want to do this. That said, I'll probably merge this branch before then.